test/e2e/config/ironic.yaml

```diff
@@ -106,24 +108,31 @@ bmoIronicUpgradeSpecs:
   initIronicKustomization: "../../ironic-deployment/overlays/e2e"
   upgradeEntityKustomization: "../../config/overlays/e2e"
   upgradeEntityName: "bmo"
-# Upgrade Ironic 24.1 -> latest | BMO 0.9
+# Upgrade BMO 0.10 -> main
 - deployIronic: true
   deployBMO: true
-  initBMOKustomization: "../../config/overlays/e2e-release-0.9"
+  initBMOKustomization: "../../config/overlays/e2e-release-0.10"
+  initIronicKustomization: "../../ironic-deployment/overlays/e2e"
+  upgradeEntityKustomization: "../../config/overlays/e2e"
+  upgradeEntityName: "bmo"
+# Upgrade Ironic 24.1 -> latest | BMO 0.10
```

Comment:
This will fail because BMO 0.10 only works with Ironic 29 or newer.
Could you make it so that we keep BMO at 0.9 and do Ironic 24.1 -> 29.0?
Same for the rest, except the last one where we can do BMO at 0.10 and Ironic 29 -> latest